compat/mingw.c

```diff
@@ -969,10 +969,12 @@ static int has_valid_directory_prefix(wchar_t *wfilename)
 	return 1;
 }
 
+static int get_reparse_point_info(const WCHAR *wpath, DWORD *ptag);
 int mingw_lstat(const char *file_name, struct stat *buf)
 {
 	WIN32_FILE_ATTRIBUTE_DATA fdata;
-	WIN32_FIND_DATAW findbuf = { 0 };
+	DWORD reparse_tag;
```

Comment:
You will need to initialize `reparse_tag` so that it can be passed to `file_attr_to_st_mode()` even if the path is not a reparse point.

Reply:
Argh! You are correct of course.

Reply:
:-) No worries. I always hope that I can help improve the patches through my reviews.

Reply:
Thanks. Somehow the compiler did not catch this, but I should have been more careful.